pmpsui/pmps.py

```diff
@@ -94,12 +101,46 @@ def __init__(self, parent=None, args=None, macros=None):
         self._channels = []
         self.setup_ui()
 
+        self.splash.finish(self)
+        self.splash = None
+
+    def setup_splash(self) -> None:
+        pixmap = QPixmap(str(Path(__file__).parent.parent / 'pmps_splash.png'))
+        self.splash = PMPSSplashScreen(pixmap)
+        self.splash.set_message_rect(QtCore.QRect(250, 100, 300, 150),
+                                     alignment=QtCore.Qt.AlignLeft)
+
+        splash_font = QtGui.QFont("Arial", 12, QtGui.QFont.Bold)
+        self.splash.setFont(splash_font)
+        self.splash.show_message('PMPS UI setup beginning...')
+        self.splash.set_progress(0)
+        self.splash.show()
```

Comment:
This ultimately shows on a different monitor than the one that loads the window on my laptop + monitor setup.

Reply:
I can move it as soon as PMPS's init finishes, though I'd the splash to appear as soon as possible